src/dungeongen.cpp

```diff
@@ -30,9 +30,8 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 
 //#define DGEN_USE_TORCHES
 
-NoiseParams nparams_dungeon_rarity(0.0, 1.0, v3f(500.0, 500.0, 500.0), 0, 2, 0.8, 2.0);
-NoiseParams nparams_dungeon_wetness(0.0, 1.0, v3f(40.0, 40.0, 40.0), 32474, 4, 1.1, 2.0);
-NoiseParams nparams_dungeon_density(0.0, 1.0, v3f(2.5, 2.5, 2.5), 0, 2, 1.4, 2.0);
+NoiseParams nparams_dungeon_density(0.9, 0.5, v3f(500.0, 500.0, 500.0), 0, 2, 0.8, 2.0);
```

Comment:
> The density noiseparam set by mgv6 is chosen to recreate the former dstribution that had a threshold of 0.2 (with offset 0 scale 1). Offset is 0.9 and scale is 0.5 to avoid causing two dungeons per chunk.

It's identical:
Currently offset 0.0 scale 1.0, threshold 0.2 for 1 dungeon.
Now the threshold for 1 dungeon is 1.0, so a match would be offset 0.8 scale 1.0, however if it went above 2.0 we would get 2 dungeons, so i halved scale to prevent this: offset 0.9 scale 0.5 which is still identical to the current threshold of 0.2.